app/models/program.py

```diff
@@ -1,6 +1,9 @@
 from app.models import*
+from app.models.term import Term
+from app.models.courseStatus import CourseStatus
 from app.models.partner import Partner
 
 class Program(baseModel):
     programName = CharField()
     partner = ForeignKeyField(Partner, null=True)
+    term = ForeignKeyField(Term)
```

Comment:
I don't think Program should have a term in it. What is the use case for this?

Reply:
We deleted it now, since we are getting information for term from the query where we join program and event.